### ISSUE TITLE
3.4.5 Headers in Basic autotable not displaying?

### ISSUE THREAD

**Issue by tobyrush** [simonbengtsson/jsPDF-AutoTable#642]
Simon, thanks for the work you've put into this. With 3.4.5, I'm suddenly not seeing column headers using the basic autotable. Here's the relevant code:

```
doc = new jsPDF({
    orientation: 'portrait',
    unit: 'pt',
    format: 'letter'
});

...

doc.autoTable(columns, rows, {
    styles: {
        fontSize: 8.5,
        textColor: 0,
        cellPadding: {bottom: 10}
    },
    headerStyles: {
        fontSize: 10,
        halign: "center",
        textColor: 255
    },
    columnStyles: {0: {fontStyle: 'bold'}},
    margin: {
        top: 50
    }
});
```

`columns` is a one-dimension array and `rows` is a two-dimension array.

It's been years since I messed with this code, so if you've changed the API feel free to tell me to go read the docs. I just thought I'd check in case it was a bug with the new version. Thank you!

**Comment by tobyrush** [comment]
I see the difference: the array I was passing was flat (`['ID','Message']`). I can format it as key/value pairs before sending to jsPDF... it was likely an error on my part initially that just happened to work!

**Comment by simonbengtsson** [comment]
Looked back at previous documentation and it seems string columns were supported! Will add support back for this and release a new version now. Thanks for reporting!

**Comment by tobyrush** [comment]
Happy to help. Thanks again for making and sharing this great library!

**Comment by simonbengtsson** [comment]
Fixed now in 3.4.6 👌

**Comment by tobyrush** [comment]
Yes, it works now! Thanks for addressing this so quickly!
